Sources/XiEditor/EditView.swift

```diff
@@ -228,6 +229,21 @@ final class EditView: NSView, NSTextInputClient, TextPlaneDelegate {
         return true
     }
 
+    override func draggingEnded(_ sender: NSDraggingInfo) {
+        guard let pasteboard = sender.draggingPasteboard().propertyList(forType: NSPasteboard.PasteboardType(rawValue: "NSFilenamesPboardType")) as? NSArray,
+            let path = pasteboard[0] as? String
```

Comment:
going for `pasteboard[0]` assumes that the array is not empty. It would be safer to go for `pasteboard.first` 😄